app/subscriber/src/features/todays-front-pages/TodaysFrontPages.tsx

```diff
@@ -85,8 +92,8 @@ export const TodaysFrontPages: React.FC = () => {
           fetchResults(data.query);
         });
       }
-    } else {
-      toast.error('FrontpageFilter setting needs to be configured.');
+    } else if (!!settings.length) {
```

Comment:
Fixed issue where error would be displayed if a user started on this page and before lookups has been initialized.